examples/src/main/scala/org/apache/spark/examples/MiniReadWriteTest.scala

```diff
@@ -59,7 +59,7 @@ object MiniReadWriteTest {
       System.exit(1)
     }
 
-    var i = 0
+    val i = 0
```

Comment:
inline this ?

Reply:
extract `val filePath = args(0)` for reuse